sbc_compassion/models/correspondence_template.py

```diff
@@ -408,24 +407,6 @@ def _compute_template_data(self):
             template.qrcode_y_max = template.page_height * float(
                 config_obj.get_param("qrcode_y_max")
             )
-            if template.pattern_image:
-                # pattern detection
-                res = pr.patternRecognition(
-                    template_cv_image,
-                    template.pattern_image,
-                    template.get_pattern_area(),
-                )
-                if res is None:
-                    raise UserError(
-                        _(
-                            "The pattern could not be detected in given "
-                            "template image."
-                        )
-                    )
-                else:
-                    pattern_keypoints = res[0]
-
-                template.nber_keypoints = pattern_keypoints.shape[0]
```

Comment:
You should also be able to remove this field from the model and all its usages (`nber_keypoints`)